src/context.ts

```diff
@@ -70,12 +72,16 @@ export async function getActionContext(): Promise<ActionContext> {
 			per_page: 100,
 		};
 
+		let queryParams = config.label
+			? { ...options, labels: config.label }
+			: options;
+
```

Comment:
I realized the label provided in the config is optional.

If it is not provided, `config.label` (as returned by `core.getInput`) will be an empty string, so the API query will default to the existing `options` variable. If `config.label` is provided, we set the `labels` field on the query.